src/gui/mainwindow.cpp

```diff
@@ -1539,10 +1539,10 @@ void MainWindow::reloadSessionStats()
                 "qBittorrent"
                 "</div>"
                 "<div style='vertical-align: baseline; height: 18px;'>"
-                "<img src=':/icons/skin/download.svg' height='14'/>&nbsp;%1"
```

Comment:
there is also download.svg in qbt-theme folder but both icons look different so had to rename it, this looks exactly like `:icon/skin/downloading.svg" but smaller in size so this name.